src/backend/resourcemanager/include/utils/simplestring.h

```diff
@@ -63,7 +63,7 @@ bool SimpleStringEmpty(SimpStringPtr str);
 int  SimpleStringLocateChar(SimpStringPtr str, char target, int *location);
 /* string to the other number values. */
 int  SimpleStringToInt32(SimpStringPtr str, int32_t *value);
-int  SimpleStringToInt64(SimpStringPtr str, int64_t *value);
+int  SimpleStringToInt64(SimpStringPtr str, int64 *value);
```

Comment:
OK. Defining int64 as "long int" is really buggy assume it means a 64bit integer. Does this comes from pg or gpdb? I do not know why we have both int64 and int64_t.